
#!/usr/bin/env python
PACKAGE = "robotics2020"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("collision_th_safe", double_t, 0, "Collision's upper threshold", 5, 0, 100)
gen.add("collision_th_crash", double_t, 0, "Collision's lower threshold", 1, 0, 100)

exit(gen.generate(PACKAGE, "collision_guard", "CollisionGuardParams"))